runtime-sdk/src/modules/core/mod.rs

```diff
@@ -116,6 +124,7 @@ pub struct Parameters {
     pub max_tx_signers: u32,
     pub max_multisig_signers: u32,
     pub gas_costs: GasCosts,
+    pub min_gas_price: u128,
```

Comment:
Should this be per denomination? E.g., a `BTreeMap<token::Denomination, u128>`, since you can also specify a denomination while paying for gas. And if a denomination is not listed, then that denomination cannot be used to pay for gas.

Reply:
Good idea, I've changed it into a map.